app/main/views/buyers.py

```diff
@@ -250,3 +250,18 @@ def seller_feedback_email(brief_id):
 
     flash('seller_feedback_email', 'info')
     return redirect(url_for('.find_buyer_by_brief_id', brief_id=brief_id))
+
+
+@main.route('/team', methods=['GET'])
+@login_required
+@role_required('admin')
+def find_team_by_team_id():
+    team_id = request.args.get('team_id')
+
+    team = data_api_client.req.admin().team(team_id).get()
+
+    return render_template_with_csrf(
+        "view_teams.html",
```

Comment:
Should this template be named `view_team` if we're displaying data for a single team?

Reply:
Made these changes.